InternetHospital.DataAccess/AppContextConfiguration/StatusConfiguration.cs

```diff
@@ -1,25 +1,30 @@
 ﻿using InternetHospital.DataAccess.Entities;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.IO;
 
 namespace InternetHospital.DataAccess.AppContextConfiguration
 {
     public class StatusConfiguration : IEntityTypeConfiguration<Status>
     {
         public void Configure(EntityTypeBuilder<Status> builder)
         {
-            const string BANNED = "Banned";
-            const string NEW = "New";
-            const string APPROVED = "Approved";
-            const string NOT_APPROVED = "Not approved";
+            const string PATH = @"..\InternetHospital.DataAccess\AppContextConfiguration\StatusConfigurationJSON.json";
 
-            builder.HasData(new Status[]
+            string jsonString = File.ReadAllText(PATH);
+
```

Comment:
The same problem with empty lines between variables.

Reply:
Done